README.md

```diff
@@ -1 +1,77 @@
 # Physlr: Construct a Physical Map from Linked Reads
+
+Physlr constructs a *de novo* physical map using linked reads from 10X Genomics or stLFR. This physical map can then be used to scaffold an existing assembly to yield chromosomal level contiguity.
+
+# Dependencies
+
+* [ntCard](https://github.com/bcgsc/ntCard)
+* [ntHits](https://github.com/bcgsc/ntHits)
+* GCC 5 or newer with openmp
+* Python 3.5 or newer and the following packages
+    * [community](https://python-louvain.readthedocs.io/en/latest/api.html)
+    * [networkx](https://networkx.github.io/)
+    * [numpy](https://numpy.org/)
+    * [scipy](https://www.scipy.org/)
+    * [sklearn](https://scikit-learn.org/stable/)
+    * [tqdm](https://tqdm.github.io/)
+
+Additionally, we recommend using pypy3 over regular python3 for speed.
+
+
+## Optional dependencies
+
+- [pigz](https://zlib.net/pigz/) for parallel gzip
+- [zsh](https://sourceforge.net/projects/zsh/) for reporting time and memory usage
+
+
+# Compiling Physlr from source
+
+```
+pip3 install --user git+https://github.com/bcgsc/physlr
+cd src
+make
```

Comment:
I believe it's possible to modify `setup.py` to run `cd src && make install`, though I've not done it myself. In the mean time, please add
```
git clone https://github.com/bcgsc/physlr
cd physlr/src && make install
```
Please add an `install` rule to the `Makefile`. It should install to the location `$(DESTDIR)$(PREFIX)`. The default value of `DESTDIR` is the empty string. The default value of `PREFIX` is `$(HOME)/.local` (or possibly `/usr/local` but I prefer `$(HOME)/.local`. Executables are installed into `$(DESTDIR)$(PREFIX)/bin`.
Within the context of `setup.py`, executables should be installed into the directory that `setup.py` tells you to install them into.

Reply:
see #118 for `install` rule. I will look into `setup.py` at another time.